engines/stark/visual/effects/bubbles.cpp

```diff
@@ -54,6 +55,8 @@ VisualEffectBubbles::~VisualEffectBubbles() {
 }
 
 void VisualEffectBubbles::render(const Common::Point &position) {
+	if (!StarkSettings->getBoolSetting(Settings::kSpecialFX)) return;
+
```

Comment:
Need to know a location with visual effects to test the correctness. Any suggestion?

Reply:
`changeLocation 40 00` => green fireflies
`changeLocation 55 00` => bubbles and fishes